omnibus/config/software/sds.rb

```diff
@@ -1,14 +1,17 @@
 name "sds"
 
-default_version "042de62f5a24fbceb4f4849256c3ee5c005b7057"
+default_version "v0.1.0"
 source git: 'https://github.com/DataDog/dd-sensitive-data-scanner'
 
 build do
     license "Apache-2.0"
     license_file "./LICENSE"
 
+    sds_version = "v0.1.0"
+
     # no Windows support for now.
     if linux_target? || osx_target?
+        command "git checkout #{sds_version}"
```

Comment:
Are you sure we actually need to explicitly checkout the version? I thought that was what `default_version` would do for you (but I can be wrong)

Reply:
I'm giving it a shot!